packages/xod-client/src/user/reducer.js

```diff
@@ -0,0 +1,20 @@
+import { UPDATE_COMPILE_LIMIT } from './actionTypes';
+
+const userReducer = (state = {}, action) => {
+  switch (action.type) {
+    case UPDATE_COMPILE_LIMIT: {
+      const limit = action.payload;
+      return {
+        ...state,
+        limit: !limit ? null : {
```

Comment:
Looks ugly 😞 
I prefer to import Ramda and use their functions to update state...